temporian/implementation/numpy/operators/window/simple_moving_average.py

```diff
@@ -0,0 +1,60 @@
+"""Implementation for the SimpleMovingAverage operator."""
+
+from typing import Dict, Optional
+
+import numpy as np
+
+from temporian.implementation.numpy.data.event import NumpyEvent, NumpyFeature
+from temporian.core.data.duration import Duration
+from temporian.core.operators.simple_moving_average import SimpleMovingAverage
+
+
+class SimpleMovingAverageNumpyImplementation:
+    """Numpy implementation for the simple moving average operator."""
+
+    def __init__(self, op: SimpleMovingAverage) -> None:
+        assert isinstance(op, SimpleMovingAverage)
+        self._op = op
+
+    def __call__(
+        self,
+        event: NumpyEvent,
+        sampling: Optional[NumpyEvent] = None,
+    ) -> Dict[str, NumpyEvent]:
+        # TODO: Add support for index.
+        if sampling is not None:
+            raise NotImplementedError("Sampling not implemented")
+
+        dst_event = NumpyEvent(data={}, sampling=event.sampling)
+
+        # For each index
+        # Note: mts => multi time series
+        # Note: ts => time series
+        for index, src_mts in event.data.items():
+            dst_mts = []
+            dst_event.data[index] = dst_mts
+            src_time = event.sampling.data[index]
+
+            # For each feature
+            for src_ts in src_mts:
+                dst_feature_name = f"sum_{src_ts.name}"

```

Comment:
Nit: `sma` prefix instead of `sum`

Reply:
Good catch.

The op is only tested manually. Ultimately, the execution code should  check the input/output of implementation execution.